test/specs/01.displayname.spec.js

```diff
@@ -0,0 +1,93 @@
+// Unit Test that are related to annotations should be here
+
+var includes = require(__dirname + '/../utils/ermrest-init.js').init();
+
+var server = includes.server;
+var ermRest = includes.ermRest;
+var ermrestUtils = includes.ermrestUtils;
+
+describe('Display Name: ', function () {
+    var catalog_id, schemaName, schema, catalog;
+
+    // This function should be present in all spec files. It will add sample database and configurations.
+    beforeAll(function (done) {
+        ermrestUtils.importData({
+            setup: require('../configuration/displayname.spec.conf.json'),
+            url: includes.url ,
+            authCookie : includes.authCookie
+        }).then(function(data) {
+            catalog_id = data.catalogId;
+            schemaName = data.schema.name;
+            console.log("Data imported with catalogId " + data.catalogId);
+            done();
+        }, function(err) {
+            console.log("Unable to import data");
+            console.dir(err);
+            done.fail();
+        });
+    });
+
+    // Test Cases:
+
+    it('schema should use its own heuristics.', function(done) {
```

Comment:
@robes I know that the descriptions that I used are too generic. But I don't want them to be very specific to the data as well. 
I changed them and the success output looks like this for now:
```
  For determining display name, 
    ✓ schema should use name styles (`underline_space`, `title_case`) that are defined in its display annotations.
    ✓ table should use its own name that is defined in display annotation.
    ✓ table should use its own name styles (`underline_space`, `title_case`) that are defined in its display annotation.
    ✓ table should use schema name styles when `name_style` and `name` are not defined in its display annotation.
    ✓ column should use its own name that is defined in display annotation.
    ✓ column should use its own name styles (`underline_space`, `title_case`) that are defined in its display annotation.
    ✓ column should use table name styles when `name_style` and `name` are not defined in its display annotation.
    ✓ column should use schema name styles when `name_style` and `name` are not defined in its display annotation nor table's display annotation.
```
Are these good enough?